discord/ui/view.py

```diff
@@ -252,9 +252,12 @@ def from_message(cls, message: Message, /, *, timeout: Optional[float] = 180.0)
             view.add_item(_component_to_item(component))
         return view
 
-    def add_item(self, item: Item[Any]) -> None:
+    def add_item(self, item: Item[Any]) -> Self:
         """Adds an item to the view.
-
+        
```

Comment:
```suggestion

```

Remove whitespace.

Reply:
> There shouldn't be any blank lines with whitespace, I suggest you run `black` on your code by installing black then typing `black discord` in a command line, or ensuring your editors formatting function uses black.

https://github.com/Rapptz/discord.py/blob/195c923becf0b691bf82786aa5773561c77cf716/discord/embeds.py#L352-L359

Looking in the embeds code there are white spaces

Reply:
The problem is you have trailing whitespace which leads to it failing CI.